examples/common/python/connectors/direct/tcs_listener/tcs_listener.py

```diff
@@ -84,6 +84,12 @@ def __init__(self, config):
 
         self.dispatcher = Dispatcher()
         rpc_methods = [
+            self.worker_registry_handler.WorkerLookUp,
+            self.worker_registry_handler.WorkerLookUpNext,
+            self.worker_registry_handler.WorkerRegister,
+            self.worker_registry_handler.WorkerSetStatus,
+            self.worker_registry_handler.WorkerRetrieve,
+            self.worker_registry_handler.WorkerUpdate,
             self.workorder_handler.WorkOrderSubmit,
             self.workorder_handler.WorkOrderGetResult,
         ]
```

Comment:
Question: Can there be multiple dispatchers attached. One for registering the workers and another for the work order?

(Probably can be taken up as a next activity to improve the performance)
I see the latter can be large number when considering the scale. If orchestration between `multiple workers for the same task` feature is added later then it could be an advantage. One should be able to add more workers and independent of the work order dispatcher pending queue.

Reply:
I think this can be achieved by implementing those handler as independent listeners, so that every listener can take it own dispatcher ~ @arsulegai 

As for multiple dispatchers within the same listener, I known no way to implement it now = =

Reply:
Right, let's make a note of this. Let us consider these thoughts in future design considerations/implementations.